### PR TITLE
Disable Take down buttons

### PR DESCRIPTION
**Pull request by Joozty**
Closes https://github.com/oacore/dashboard/issues/96

### CODE REVIEW

**Comment by Joozty** [comment]
Ohh sorry for the misunderstanding. It should be fine now.

File: components/published-toggle/index.css [hunk 1]
```diff
@@ -5,11 +5,12 @@
   --toggle-active-color: #5a9216;
   --toggle-switch-border-radius: 2px;
 }
+
 .toggle {
   position: relative;
   display: inline-block;
-  cursor: pointer;
```
Comment (viktor-yakubiv) [inline components/published-toggle/index.css]: Let's keep it.

File: components/published-toggle/index.jsx [hunk 2]
```diff
@@ -1,13 +1,12 @@
-import React, { useState } from 'react'
+import React from 'react'
 import classNames from 'classnames'
 
 import styles from './index.css'
 
 const PublishedToggle = ({ className, defaultVisibility, isExpanded }) => {
```
Comment (viktor-yakubiv) [inline components/published-toggle/index.jsx]: Could you please provide `disabled` prop here and just disable it from parent?

Disabled toggle should have transparent background and different cursor.

```css
.disabled {
  opacity: 0.5; /* or any other */
  cursor: not-allowed;
}
```

File: components/published-toggle/index.jsx [hunk 3]
```diff
@@ -14,22 +19,29 @@ const PublishedToggle = ({ className, defaultVisibility, isExpanded }) => {
...
-          [styles.expanded]: isExpanded,
-        })}
+        className={classNames
+          .use('toggle-checkbox', {
+            expanded: isExpanded,
+          })
+          .from(styles)}
         type="checkbox"
         onChange={event => {
+          if (disabled) return
```
Comment (viktor-yakubiv) [inline components/published-toggle/index.jsx]:
```suggestion
          if (disabled) {
            event.preventDefault()
            return
          }
```

**Pull request merged**